app/models/html_file.rb

```diff
@@ -325,16 +325,16 @@ def validate(record)
 end
 
 class HtmlFile < ApplicationRecord
-  include Ensure_docx_content_type # fixing docx content-type detection problem, per https://github.com/thoughtbot/paperclip/issues/1713
+  include EnsureDocxContentType # fixing docx content-type detection problem, per https://github.com/thoughtbot/paperclip/issues/1713
```

Comment:
New classloader requires filename to match class name (class name should be in CamelCase, and filename in snake_case)